From 8b5b843fcbc3e03e0fc45f3caf8cf5fc477e8613 Mon Sep 17 00:00:00 2001
From: Jakub Zawadzki <darkjames-ws@darkjames.pl>
Date: Mon, 25 Sep 2017 18:10:01 +0200
Subject: [PATCH] udpdump: Add missing include for timeval struct.

struct timeval is defined in <sys/time.h>, include it.

It should fix building with musl:

udpdump.c: In function 'setup_listener':
udpdump.c:126:9: error: variable 'timeout' has initializer but incomplete type
  struct timeval timeout = { 1, 0 };

Change-Id: Id9be579e50ea2845ca609708e8ec4d1a3858ecc0
Reviewed-on: https://code.wireshark.org/review/23728
Petri-Dish: Jakub Zawadzki <darkjames-ws@darkjames.pl>
Tested-by: Petri Dish Buildbot <buildbot-no-reply@wireshark.org>
Reviewed-by: Jakub Zawadzki <darkjames-ws@darkjames.pl>
---

diff --git a/extcap/udpdump.c b/extcap/udpdump.c
index f26a5ff..907c896 100644
--- a/extcap/udpdump.c
+++ b/extcap/udpdump.c
@@ -31,6 +31,10 @@
 #include <glib/gprintf.h>
 #include <stdlib.h>
 
+#ifdef HAVE_SYS_TIME_H
+	#include <sys/time.h>
+#endif
+
 #ifdef HAVE_SYS_SOCKET_H
 	#include <sys/socket.h>
 #endif
